File: frontend/src/app/directory-tech/shared/pagination.ts
export type PaginationOptions = Partial<{
  pageNum: number;
  pageSize: number;
}>;

export type Pagination = {
  pagination: {
    pageIndex: number;
    total: number;
    records: number;
  };
};

export const PAGE_NUM = 1;
export const PAGE_SIZE = 50;

File: Backend/BioTonFMS.Telematica/Dtos/Pagination.cs
namespace BioTonFMS.Telematica.Dtos
{
    /// <summary>
    /// Параметры постраничного вывода
    /// </summary>
    public class Pagination
    {
        /// <summary>
        /// Номер текущей страницы
        /// </summary>
        public int PageIndex { get; set; }

        /// <summary>
        /// Общее количество страниц для вывода всех данных
        /// </summary>
        public int Total { get; set; }

        /// <summary>
        /// Общее количество записей
        /// </summary>
        public int Records { get; set; }
    }
}
